services/hexathons/src/routes/hexathon-users.ts

```diff
@@ -78,6 +78,31 @@ hexathonUserRouter.route("/:hexathonId/users/:userId").patch(
   })
 );
 
+hexathonUserRouter.route("/:hexathonId/users/profile/:userId").patch(
```

Comment:
can you change the route to `/:hexathonId/users/:userId/profile` just to keep it consistent with our other endpoints?